css/danti.css

```diff
@@ -0,0 +1,306 @@
+.contener{
```

Comment:
If you do follow my other comment and include `style.css` on the store page, be aware that you have a rulset in that CSS file for `.contener` as well. If you want them to do different things, you'll need to give them different class names.